sdk/python/jobs/pipelines/1h_automl_in_pipeline/automl-forecasting-in-pipeline/automl-forecasting-in-pipeline.ipynb

```diff
@@ -201,19 +453,388 @@
+    "            f\"--output_dataset_name {output_ds_name}\"\n",
+    "        ),\n",
+    "        environment=env,\n",
+    "    )\n",
+    "\n",
+    "    call_inferencing = inference_func(test_dataset=forecasting_inference_data)\n",
+    "\n",
+    "    return {\"output_dataset\": call_inferencing.outputs.output_dataset}\n",
+    "\n",
+    "\n",
+    "pipeline_job_data = automl_forecasting(\n",
+    "    forecasting_inference_data=test_dataset,\n",
+    ")\n",
+    "\n",
+    "# set pipeline level compute\n",
+    "pipeline_job_data.settings.default_compute = cluster_name\n",
+    "\n",
+    "pipeline_job_data.outputs.output_dataset.mode = \"rw_mount\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "inference_job = ml_client.jobs.create_or_update(\n",
+    "    pipeline_job_data, experiment_name=\"pipeline_inference\"\n",
+    ")\n",
+    "inference_job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.stream(inference_job.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.3 Get the predicted data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.download(\n",
+    "    inference_job.name, download_path=\".\", output_name=\"output_dataset\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "inference_df = pd.read_csv(\n",
+    "    os.path.join(\"named-outputs\", \"output_dataset\", f\"{output_ds_name}.csv\"),\n",
+    "    parse_dates=[time_column_name],\n",
+    ")\n",
+    "inference_df.tail(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 4. Schedule Pipeline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This section is about how to schedule a pipeline for periodically predictions. For more info about pipeline schedule and pipeline endpoint, please follow this [notebook](https://github.com/Azure/azureml-examples/blob/83c67ec408f10e2e07b3a2a3e648023caa09e112/sdk/python/schedules/job-schedule.ipynb).<br>\n",
+    "## 4.1. Define a schedule\n",
+    "If `test_dataset` is going to refresh every 4 weeks before Friday 16:00 and we want to predict every 4 weeks (forecast_horizon), we can schedule our pipeline to run every 4 weeks at 16:00 to get daily inference results. You can refresh your test dataset (a newer version will be created) periodically when new data is available (i.e. target column in test dataset would have values in the beginning as context data, and followed by NaNs to be predicted). The inference pipeline will pick up context to further improve the forecast accuracy."
```

Comment:
Perhaps we can say "If `test_dataset` will be updated every 4 weeks on Friday 16:00 and the objective is to generate a 4 week (forecast_horizon) forecast, ..."

Reply:
Fixed